charts/studio/templates/configmap.yaml

```diff
@@ -1,9 +1,87 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: studio-ca-certificates
+  name: studio
 data:
-  {{- if .Values.global.customCaCert }}
-  self_signed_ca.crt: |
-{{ .Values.global.customCaCert | indent 4}}
+  ALLOWED_HOSTS: "*"
+{{/*  {{- if .Values.studioBackend.ingress.enabled }}*/}}
+{{/*  {{- range $host := .Values.studioBackend.ingress.hosts }}*/}}
+{{/*    {{- range .paths }}*/}}
+{{/*  #*/}}
+{{/*  API_URL: "http{{ if $.Values.studioBackend.ingress.tls }}s{{ end }}://{{ $host.host }}{{ .path }}"*/}}
+{{/*    {{- end }}*/}}
+{{/*  {{- end }}*/}}
+{{/*  {{- else }}*/}}
+  API_URL: "http://studio-backend.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.studioBackend.service.port }}"
```

Comment:
This workflow for enabled Ingress is commented because of a bug https://github.com/iterative/helm-charts/issues/41

Reply:
It is now more broken than before though, for one path it works now, and this will break this (and prevent me from testing it if this gets merged. Can you move the logic here as a first stage and we'll refine the paths list handling afterwards please?

Reply:
Thanks @omesser, it is outdated now. I migrated it from the changes made https://github.com/iterative/helm-charts/pull/49